integration-tests/create-react-app/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "create-react-app",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "react": "^15.6.1",
+    "react-dom": "^15.6.1"
+  },
+  "optionalDependencies": {
+    "react-scripts": "1.0.11"
```

Comment:
Sorry, missed a few comments here. It's not super hard, mostly it means the example projects in this repo start to look a little funky because a lot of common packages require Node 6+ now (e.g. `react-match-media` and `create-react-app`), so we have to do stuff like mark them as optional dependencies. I updated this repo to target Node 6+, but can certainly adjust it back to 4 if it's needed!